### PR TITLE
Bttfclock

### PR DESCRIPTION
**Pull request by NoobEjby**
My first watchface and code for the bangle.js v2.

Please let me know if something seems wrong

### CODE REVIEW

File: apps/bttfclock/BackToTheFuture.app.js
Comment (thyttan) [inline apps/bttfclock/BackToTheFuture.app.js:1]: Rename file to "app.js".

File: apps/bttfclock/ChangeLog.log
Comment (thyttan) [inline apps/bttfclock/ChangeLog.log:1]: Rename file to "ChangeLog".

File: apps/bttfclock/bttf_clock_icon.js
Comment (thyttan) [inline apps/bttfclock/bttf_clock_icon.js:1]: rename to "app-icon.js"

File: apps/bttfclock/bttf_icon.png
Comment (thyttan) [inline apps/bttfclock/bttf_icon.png:1]: rename to "app.png"

File: apps/bttfclock/bttf_screenshot.png
Comment (thyttan) [inline apps/bttfclock/bttf_screenshot.png:1]: rename to "screenshot.png".

File: apps/bttfclock/metadata.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "id": "bttfclock",
+  "name": "Back To The Future",
+  "version": "0.01",
+  "description": "My favorit Garmin watchface",
+  "icon": "bttf_clock.png",
```
Comment (thyttan) [inline apps/bttfclock/metadata.json:6]:
```suggestion
  "icon": "app.png",
```

File: apps/bttfclock/metadata.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  "screenshots": [{"url":"bttf_screenshot.png"}],
+  "type": "clock",
+  "tags": "clock",
+  "supports": ["BANGLEJS2"],
+  "allow_emulator": true,
+  "storage": [
+    {"name":"BackToTheFuture.app.js","url":"BackToTheFuture.app.js"},
+    {"name":"bttfclock.img","url":"bttf_clock_icon.js","evaluate":true}
```
Comment (thyttan) [inline apps/bttfclock/metadata.json:14]:
```suggestion
    {"name":"bttfclock.img","url":"app-icon.js","evaluate":true}
```

**Review by thyttan — COMMENTED** [review]
Thanks! Added some review comments.

**Comment by NoobEjby** [comment]
Thanks @thyttan I saw the error in the flow, but it would have taken me a little longer to figure out without your help. I have made the changes and am crossing my fingers for it to work now :)

**Comment by thyttan** [comment]
Nice! The linter still complains about three variables that are never used. Either remove them or comment them out and I think we should be good to go :+1:

**Comment by thyttan** [comment]
Merging - thanks again!

**Comment by NoobEjby** [comment]
Thanks for the help

**Pull request merged**